features/performance/test/machineconfig_test.go

```diff
@@ -31,11 +36,29 @@ var _ = Describe("TestPerformanceMachineConfig", func() {
 			}
 		})
 	})
+
+	Context("RT Kernel setup MachineConfig", func() {
+		It("Should ship a correctly encoded payload", func() {
+			mc := loadMC(machineconfigRTKernelYaml)
+			Expect(len(mc.Spec.Config.Storage.Files)).To(BeNumerically(">=", 1))
+			for _, encodedFile := range mc.Spec.Config.Storage.Files {
+				validateContentSource(encodedFile.Contents.Source)
+			}
+		})
+		It("Should ship at least an enabled systemd unit", func() {
+			mc := loadMC(machineconfigRTKernelYaml)
+			Expect(len(mc.Spec.Config.Systemd.Units)).To(BeNumerically(">=", 1))
+			for _, unitFile := range mc.Spec.Config.Systemd.Units {
+				Expect(*unitFile.Enabled).To(BeTrue())
```

Comment:
I need to relearn golang, but the asterisk here is suspicious :) Why do the two lines differ wrt to it?

Reply:
Enabled is a boolean pointer.

to be slightly more accurate, we'd probably want this

```
Expect(unitFile.Enabled).ToNot(BeNil())
Expect(*unitFile.Enabled).To(BeTrue())
```

Reply:
@davidvossel yep, the more accurate the better. Changing.

Reply:
Ah, makes sense now. Thanks